forms/common-themes/src/main/resources/theme/base/admin/resources/js/controllers/users.js

```diff
@@ -644,6 +644,16 @@ module.controller('GenericUserFederationCtrl', function($scope, $location, Notif
                 instance.config.updateProfileFirstLogin = true;
                 instance.config.allowKerberosAuthentication = true;
             }
+
+            if (providerFactory.properties) {
+
+                for (var i = 0; i < providerFactory.properties.length; i++) {
+                    var configProperty = providerFactory.properties[i];
+                    var configValue = configProperty.type == "boolean" ? (configProperty.defaultValue === true ? 'true' : 'false') : configProperty.defaultValue;
```

Comment:
Is there a better way to do this? it seems that the "on/off" toggles need the boolean as a string to function properly.